test/jdk/java/jextract/SmokeTest.java

```diff
@@ -24,7 +24,7 @@
 /*
  * @test
  * @build JextractApiTestBase
- * @run testng/othervm -Dforeign.restricted=permit SmokeTest
+ * @run testng/othervm --enable-native-access=jdk.incubator.jextract SmokeTest
```

Comment:
This makes me wonder if jextract should be white-listed. After all it's JDK-provided, so all usages of restricted API should be correct, and if there's a crash in the jextract tool, well, that's a bug in the tool, no?